### PR TITLE
 Fix #6533 - Add simple telemetry

### LINKED ISSUE #6533: Add simple telemetry events
Depends on issue #6529

### Example

```js
import { recordEvent } from "src/utils/telemetry"; // Correct the path as appropriate

// Event without extra properties
recordEvent("add_breakpoint");
```

### Events to be added

- [ ]  "continue"  https://bugzil.la/1463122
- [ ]  "add_breakpoint"  https://bugzil.la/1463123
- [ ]  "remove_breakpoint"  https://bugzil.la/1463124
- [ ]  "pretty_print"  https://bugzil.la/1463125
- [ ]  "blackbox"  https://bugzil.la/1463126

Added (via @digitarald):

- [ ] "open_file" https://bugzil.la/1463108

### PR DESCRIPTION
**Pull request by darkwing**
Fixes Issue: #6533

### Events to be Added

- [x]  "continue"  https://bugzil.la/1463122
- [x]  "add_breakpoint"  https://bugzil.la/1463123
- [x]  "remove_breakpoint"  https://bugzil.la/1463124
- [x]  "pretty_print"  https://bugzil.la/1463125
- [x]  "blackbox"  https://bugzil.la/1463126

### CODE REVIEW

File: src/actions/sources/blackbox.js [hunk 1]
```diff
@@ -17,6 +19,10 @@ export function toggleBlackBox(source: Source) {
   return async ({ dispatch, getState, client, sourceMaps }: ThunkArgs) => {
     const { isBlackBoxed, id } = source;
 
+    if (!isBlackBoxed) {
+      recordEvent("blackbox");
```
Comment (darkwing) [inline src/actions/sources/blackbox.js]: Only recording when a source is not yet blackboxed.

File: src/actions/sources/prettyPrint.js [hunk 2]
```diff
@@ -79,6 +80,10 @@ export function togglePrettyPrint(sourceId: string) {
       return {};
     }
 
+    if (!source.isPrettyPrinted) {
+      recordEvent("pretty_print");
```
Comment (darkwing) [inline src/actions/sources/prettyPrint.js]: Only recording when a source is not a prettyprint source.

File: src/actions/breakpoints/tests/breakpoints.spec.js [hunk 3]
```diff
@@ -25,6 +28,9 @@ describe("breakpoints", () => {
       line: 2,
       sourceUrl: "http://localhost:8000/examples/a"
     };
+
+    setupHelper({ selectors: {} });
```
Comment (jasonLaster) [inline src/actions/breakpoints/tests/breakpoints.spec.js]: lets do this in `tests/tests-setup` beforeEach

Comment (darkwing) [reply src/actions/breakpoints/tests/breakpoints.spec.js]: Good shout.

**Comment by jasonLaster** [comment]
@darkwing i just pushed  a small change to prevent logging the dbg stuff when testing

screen shot 2018-07-24 at 3 30 11 pm

**Pull request merged**